runtime/lua/vim/snippet.lua

```diff
@@ -1,13 +1,14 @@
 local G = vim.lsp._snippet_grammar
-local snippet_group = vim.api.nvim_create_augroup('vim/snippet', {})
-local snippet_ns = vim.api.nvim_create_namespace('vim/snippet')
+local api = vim.api
```

Comment:
This is just adding noise with noticeable benefit.

Reply:
onelese table lookup . it's somewhat trivial.

Reply:
Yes, it's trivial, and the table lookup is not a bottleneck in this code (unless benchmarks say otherwise).

And now I have to wade through a long diff to find the ones that make the _actual_ change described in the PR.